src/main/resources/mockdata/05bdf3c8-01f0-4ddb-bd6c-6efd465f9e33.json

```diff
@@ -1,154 +1,172 @@
-          "alert": ""
+          "id": "07v5d3e6-cvbe-l1e8-a8d5-f2801f1b9fd12",
+          "alert": "",
+          "po_line_id": "07c5d04e-c123-1132-87d5-f2801f1b9fc1"
         }
       ],
+      "cancellation_restriction": false,
+      "cancellation_restriction_note": "",
       "claims": [
         {
-          "id": "07c603de-cabe-11e8-a8d5-f2801f1b9fd1",
-          "claim": true,
-          "sent": "",
-          "grace": 60,
-          "po_line_id": null
+          "id": "07cld666-caxe-11e8-a8d5-f2801f1b9fd12",
+          "claimed": true,
+          "sent": null,
+          "grace": 0,
+          "po_line_id": "07c5d04e-c123-1132-87d5-f2801f1b9fc1"
+        }
+      ],
+      "collection": false,
+      "contributors": [
+        {
+          "contributor": "",
+          "contributor_type": ""
         }
       ],
       "cost": {
-        "id": "07c60802-cabe-11e8-a8d5-f2801f1b9fd1",
+        "id": "07c5d79v-cabe-11e8-a8d5-f22801f1b9fd1",
         "list_price": 500.00,
         "currency": "USD",
-        "quantity": 2,
-        "estimated_price": 500.00
+        "quantity_physical": 1,
+        "quantity_electronic": 1,
+        "po_line_estimated_price": 500.00,
+        "po_line_id": "07c5d04e-c123-1132-87d5-f2801f1b9fc1"
       },
+      "description": "",
       "details": {
-        "id": "07c60942-cabe-11e8-a8d5-f2801f1b9fd1",
-        "title": "Acquisitions Rules the Library",
+        "id": "b7c5d792-cabe-1le29-a8d5-f2801f1b9fd1",
         "receiving_note": "",
-        "product_id": "12357",
-        "subscription_from": "2018-07-20T10:00:00.000+0000",
-        "material_type": "DVD+streaming",
+        "product_ids": [
+          {
+            "product_id": "12357",
+            "product_id_type": "Vendor Title Number"
+          }
+        ],
+        "material_types": [
+          "DVD+streaming"
+        ],
+        "subscription_from": "2018-07-20T00:00:00.000Z",
         "subscription_interval": 1095,
-        "subscription_to": "2021-07-19T10:00:00.000+0000",
-        "description": ""
+        "subscription_to": "2021-07-19T00:00:00.000Z",
+        "po_line_id": "07c5d04e-c123-1132-87d5-f2801f1b9fc1"
       },
+      "donor": "",
       "eresource": {
-        "id": "07c60a6e-cabe-11e8-a8d5-f2801f1b9fd1",
-        "activation_status": "y",
+        "id": "07c5g8be-cnbe-11e8-a8d5-f2801f1b9fd1",
+        "activated": true,
         "activation_due": 1,
         "create_inventory": true,
         "trial": false,
-        "expected_activation": "2018-07-20T10:00:00.000+0000",
+        "expected_activation": "2019-07-20T00:00:00.000Z",
         "user_limit": 0,
-        "material_id": null,
-        "access_provider": "Ro*Co films"
-      },
-      "license": {
-        "id": "07c60d70-cabe-11e8-a8d5-f2801f1b9fd1",
-        "code": "8b23",
-        "description": "",
-        "reference": ""
+        "access_provider": "Ro*Co films",
+        "po_line_id": "07c5d04e-c123-1132-87d5-f801f1b9fd1",
```

Comment:
All of the po_line_id fields in a given po_line should be set to the id of that po_line.  This may seem redundant (in fact it is), but it's there since these sub-objects are stored separately from the po_line in the database, and there's a desire to make them doubly linked, e.g. po_line <-> obj